iXBRLViewerPlugin/viewer/src/js/search.js

```diff
@@ -65,6 +65,16 @@ export class ReportSearch {
             }
         }
         const builder = new lunr.Builder();
+        builder.pipeline.add(
+          lunr.trimmer,
+          lunr.stopWordFilter,
+          lunr.stemmer
+        )
+
+        builder.searchPipeline.add(
+          lunr.stemmer
```

Comment:
Any particular reason not to also use the trimmer and stop word pipeline functions for search?

Reply:
I'm not sure.  This is just replicating the default behaviour that you get if you use the `lunr()` convenience constructor (see https://github.com/olivernn/lunr.js/blob/master/lunr.js#L43) which is what we were using before this PR.  I assume the idea is that people wouldn't use punctuation in their search queries.  

We had to move away from the convenience constructor because it uses a callback to populate the index, and we can't `yield` from within the callback.